
   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

  2024-06-27 03:42:01 App\Jobs\WriteLog .............................. RUNNING
  2024-06-27 03:42:01 App\Jobs\WriteLog .......................... 4.07ms DONE
  2024-06-27 03:43:47 App\Jobs\WriteLog .............................. RUNNING
  2024-06-27 03:43:47 App\Jobs\WriteLog .......................... 1.59ms DONE
  2024-06-27 03:44:05 App\Jobs\WriteLog .............................. RUNNING
  2024-06-27 03:44:05 App\Jobs\WriteLog .......................... 1.39ms DONE

   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Error while reading line from the server. [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+27 vendor frames [22m

  28  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))


   Predis\Connection\ConnectionException 

  Network is unreachable [tcp://host.docker.internal:6379]

  at vendor/predis/predis/src/Connection/AbstractConnection.php:144
    140â–•      */
    141â–•     protected function onConnectionError($message, $code = 0)
    142â–•     {
    143â–•         CommunicationException::handle(
  âžœ 144â–•             new ConnectionException($this, "$message [{$this->getParameters()}]", $code)
    145â–•         );
    146â–•     }
    147â–• 
    148â–•     /**

      [2m+31 vendor frames [22m

  32  artisan:35
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))

